packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -2318,6 +2319,7 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
         invertSearchColumns
       );
       this.setState({ searchFilter });
+      this.stopLoading();
```

Comment:
I'm assuming you'll merge the search change first and then rebase this one, but just so this doesn't actually get merged this shouldn't be here.